lib/sphinx_rtm/messages.ex

```diff
@@ -1,7 +1,12 @@
 defmodule SphinxRtm.Messages do
-  alias Sphinx.SlackUtils
-  alias Sphinx.Riddles
   alias SphinxRtm.Messages.Parser
+  alias Sphinx.Riddles
+  alias Sphinx.SlackUtils
+  alias Sphinx.SlackArgs
+
+  @user_token Application.get_env(:slack, :user_token)
+  @slack_url "https://slack.com/api/"
+  @channel "CR1LTP79B"
```

Comment:
Hmm, I understand since we want to limit the use of the bot in 1 channel for now, but I don't know if it's a good idea to hard-code the channel-id here 🤔 In a way, we could you %message.channel and search inside that channel, it would give more flexibility and if someone else to use the code on different workspace, it would still work. However, I'm aware of the consequence that if someone chat directly with sphinx, they can't really search in other channel 😕

Reply:
Just a thought here because I don't have a good solution thought out xD

Reply:
totally agree, as it still working in progress I'll clean all